why-use-php.php

```diff
@@ -0,0 +1,402 @@
+<?php
+$_SERVER['BASE_PAGE'] = 'why-use-php.php';
+include_once __DIR__ . '/include/prepend.inc';
+site_header('Why Use PHP? Fast, Flexible, Powerful Web Development', [
+    'current' => 'community',
+    'description' => 'PHP is the ideal choice for web development. Learn about its speed, flexibility, and rich ecosystem. Explore PHP\'s capabilities in building websites, APIs, e-commerce platforms, and more. See why industry leaders trust PHP for their projects.',
+    'css' => ['why-use-php.css'],
+]);
+?>
+
+<div class="block block--grid">
+    <div class="block__intro">
+        <div class="title">Why Use PHP?</div>
+
+        <div class="hero">
+            <div class="subtitle">Fast, Flexible, Pragmatic. Everywhere.</div>
+            <div class="description">
+                <p>PHP is a general-purpose language that excels on the web.</p>
+                <p>Here are some compelling reasons to choose PHP:</p>
+            </div>
+            <ul>
+                <li>Versatile and widely adopted</li>
+                <li>Extensive ecosystem of libraries and frameworks</li>
+                <li>Excellent performance and scalability</li>
+                <li>Strong community support</li>
+                <li>Cost-effective development</li>
+            </ul>
+
+            <div class="subtitle">With PHP You Can Do More and Deliver Faster!</div>
+        </div>
+    </div>
+    <div class="block__body">
+        <?php highlight_php_trimmed(
+            <<<'PHP'
+class HomeController
+{
+    #[Route(path: '/', method: HttpMethod::GET)]
+    public function index(): Response
+    {
+        return new Response(
+            headers: ['Content-Type' => 'text/html'],
+            body: '<h1>Welcome to the Modern PHP Application!</h1>'
```

Comment:
```suggestion
            body: '<h1>Welcome to Modern PHP!</h1>'
```

Not sure what the best exact wording is, but I believe the current sentence is incorrect, right?